### ISSUE TITLE
404 page enhancement

### ISSUE THREAD

**Issue by Mayank8528** [kanak227/NGO-WEBSITE#154]
404 page can be enhanced as it lacks aria labels, and color contrast can bee also improved also the CSS timings can be standardized. 

ASSIGN ME THIS

**Comment by kanak227** [comment]
Go ahead @Mayank8528
